src/_data/sites/hi.yml

```diff
@@ -0,0 +1,521 @@
+    bronze:
+      title: Bronze प्रायोजक
+      description: >
+        $200 USD के मासिक दान के साथ Silver प्रायोजक बनें (4 घंटे के रखरखाव और विकास का समर्थन)।
+        Gold प्रायोजक के logo को ESLint README और eslint.org के शीर्ष पृष्ठ पर पाया जा सकता है।
+    technology:
+      title: Technology प्रायोजक
+      description: >
+        ओपन सोर्स इकोसिस्टम और हमारे काम में हमारे योगदान के हिस्से के रूप में technology प्रायोजक हमारे उत्पादों और सेवाओं को मुफ्त में लाइसेंस देते हैं।
+    backers:
+      title: सभी समर्थक
+      description: >
+        ESLint को बनाए रखने में सहायता के लिए समर्थक मासिक योगदान प्रदान करते हैं।
+
+  recent_donations:
+    title: हाल ही में एकमुश्त दान
+    description: >
+      हमें अद्भुत लोगों से बहुत से अस्थायी दान भी मिलते हैं । यहां OpenCollective और GitHub से कुछ हालिया दान दिए गए हैं।
+
+  sponsor_title: SPONSOR_NAME हर महीने AMOUNT का दान दे रहा है
+  tech_sponsor_title: TECH provided by SPONSOR_NAME
+
+#------------------------------------------------------------------------------
+# Donate Page
+#------------------------------------------------------------------------------
+
+donate_page:
+  title: दान करें
+  description: >
+    ESLint चल रहे रखरखाव और विकास के लिए दान पर निर्भर करता है।
+     वर्तमान में BACKER_COUNT कंपनियां, संगठन और व्यक्ति 
+     हमारी मदद करने के लिए हर महीने MONTHLY_DONATIONS दान कर रहे हैं।
+  hero:
+    cta:
+      text: हमारे प्रायोजक देखें
+      url: /sponsors
+  donation_options:
+    title: दान करने के तरीके
+    subtitle: हम निम्नलिखित साइटों के माध्यम से दान स्वीकार करते हैं।
+    open_collective:
+      title: Open Collective
+      subtitle: निगमों के लिए सबसे लोकप्रिय योजना
+      cta:
+        text: Open Collective के माध्यम से योगदान करें
+      items:
+        - text: 501(c)(3) गैर-लाभकारी संस्थाओं को प्रत्यक्ष दान
+        - text: अस्थायी और नियमित दान
+        - text: आप क्रेडिट कार्ड, बैंक हस्तांतरण, PayPal द्वारा भुगतान कर सकते हैं
+        - text: एक ओपन सोर्स फंड लॉन्च करके कई परियोजनाओं के लिए दान संभव है
+        - text: अधिकांश देशों में उपलब्ध
+    github_sponsors:
+      title: GitHub Sponsors
+      subtitle: उन्नत सुविधाएँ और रिपोर्ट
+      cta:
+        text: GitHub के माध्यम से योगदान करें
+      items:
+        - text: GitHub को भुगतान द्वारा दान
+        - text: अस्थायी और नियमित दान
+        - text: आप क्रेडिट कार्ड से भुगतान कर सकते हैं या मौजूदा GitHub बिलिंग विधियों (चालान सहित) का उपयोग कर सकते हैं
+        - text: अपने GitHub प्रोफ़ाइल पर "sponsor" बैज दिखाएं
+        - text: अपने मौजूदा GitHub लॉगिन का उपयोग करें
+  budget:
+    title: धन का उपयोग
+    subtitle: >
+      ESLint परियोजना, परियोजना और ESLint समुदाय को समग्र रूप से लाभान्वित करने के लिए दान किए गए धन का उपयोग करती है।
+      सामान्य तौर पर, हम हर महीने निम्नलिखित क्षेत्रों पर पैसा खर्च करते हैं।
+    items:
+      - name: टीम विकास
+        text: >
+          We pay our team members an hourly rate based on their seniority, currently HOURLY_RATE_TSC USD/hour
```

Comment:
Let's also keep this in hindi since this is not a privacy related. WDYS?

Reply:
Yeah, I suppose I forgot to translate this line.